CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+### Features
+
+# [1.3.0](https://github.com/princjef/irontask/compare/v1.2.0...v1.3.0) (2020-09-10)
```

Comment:
please revert the changes in this file. the build system will automatically populate this from your commit message